### ISSUE TITLE
Support installation on Kubernetes

### ISSUE THREAD

**Issue by davidstanke** [GoogleCloudPlatform/fourkeys#201]
Many users have asked for the ability to install Four Keys onto Kubernetes. This requires (at least):

- Use some replacement for Pub/Sub.  Maybe an open source pub/sub tool running in Kubernetes?
- Run the event-handler and data-parsers in Kubernetes.  These are already container based images so they should fit in nicely.  
- Replace BigQuery with a vanilla SQL database running Kubernetes
- Update the API calls in the services to use the open source equivalents of Pub/Sub and BigQuery.

**Comment by emirot** [comment]
It would be great to see that project run on other stack as well.

For K8s what were you thinking:
- Rabbitmq, kafka
- event handler and parser will just be deployment
- big query could be using Druid

**Comment by davidstanke** [comment]
Yes, that's in line with what I was thinking, @emirot -- would you be interested in collaborating on this?

**Comment by irab** [comment]
Apache Pulsar could be a good fit for a pub/sub replacement? There's a helm chart for easy deployment here:
https://pulsar.apache.org/docs/en/helm-deploy/

There's [an open PR for Pulsar integration with Druid](https://github.com/apache/druid/pull/11224) that just need a bit of work. The built in Trino (formerly Presto) SQL interface supported by [Pulsar isn't supported by Grafana](https://grafana.com/grafana/plugins/?type=datasource).

**Comment by treyhyde** [comment]
Being that this project already uses Knative (CloudRun), Knative Eventing would be a solid choice.
